client/app/collections/search.coffee

```diff
@@ -17,12 +17,11 @@ module.exports = class Search extends Filtered
 
     filter: (model) =>
         return true unless @search
-
-        filters = @search.match /(\w+):(\w+)/gi
+        filters = @search.match /(\w+:[^\s]+)/gi
         filters.reduce (memo, filter) ->
```

Comment:
@frankrousseau I agree and reduce is more performant (and does the tricks of gaining a subset of models that matches)